lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -14,14 +14,23 @@ class LoggerMiddleware < Bricks::Middleware
       def call(params)
         params = params.to_hash
         logger = nil
+
         if params['GDC_LOGGING_OFF']
           logger = NilLogger.new
         else
           logger = params[:GDC_LOGGER_FILE].nil? ? Logger.new(STDOUT) : Logger.new(params[:GDC_LOGGER_FILE])
           logger.level = params['GDC_LOG_LEVEL'] if params['GDC_LOG_LEVEL']
           logger.info('Pipeline starts')
+
+          original_formatter = Logger::Formatter.new
+          logger.formatter = proc { |severity, datetime, progname, msg, to_splunk=params['SPLUNK_LOG_ALL']|
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.